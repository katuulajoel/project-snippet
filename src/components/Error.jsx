import PropTypes from "prop-types";
import React from "react";

import Icon from "./Icon";

const Error = ({ message }) => {
  return (
    <div className="alert alert-danger">
      <Icon name="attention" /> {message || ""}
    </div>
  );
};

Error.propTypes = {
  message: PropTypes.string,
};

export default Error;
