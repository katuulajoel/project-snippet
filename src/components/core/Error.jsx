/* eslint-disable react/prop-types */
import React from "react";

import Icon from "./Icon";

const Error = ({ message }) => {
  return (
    <div className="alert alert-danger">
      <Icon name="attention" /> {message || ""}
    </div>
  );
};

export default Error;
