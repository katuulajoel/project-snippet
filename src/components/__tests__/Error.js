import React from "react";
import renderer from "react-test-renderer";
import Error from "../Error";

describe("Auth layout test", () => {
  it("Snapshot test for Login component", () => {
    const tree = renderer.create(<Error />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
