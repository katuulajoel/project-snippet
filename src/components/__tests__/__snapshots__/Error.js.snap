// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth layout test Snapshot test for Login component 1`] = `
<div
  className="alert alert-danger"
>
  <i
    className="tg-ic-attention  "
  />
   
  
</div>
`;
